website/source/guides/acl.html.markdown

```diff
@@ -0,0 +1,310 @@
+| [operator](#operator-rules) | Cluster-level operations in the Operator API |
+
+Constructing rules from these policies is covered in detail in the Rule Specification section below.
+
+### Multi-Region Configuration
+
+Nomad supports multi-datacenter and multi-region configurations. A single region is able to service multiple datacenters, and all servers in a region replicate their state between each other. In a multi-region configuration, there is a set of servers per region. Each region operates independently and is loosely coupled to allow jobs to be scheduled in any region and requests to flow transparently to the correct region.
+
+When ACLs are enabled, Nomad depends on an "authoritative region" to act as a single source of truth for ACL policies and global ACL tokens. The authoritative region is configured in the `server` stanza of agents, and all regions must share a single a single authoritative source. Any ACL policies or global ACL tokens are created in the authoritative region first. All other regions replicate ACL policies and global ACL tokens to act as local mirrors. This allows policies to be administered centrally, and for enforcement to be local to each region for low latency.
+
+Global ACL tokens are used to allow cross region requests. Standard ACL tokens are created in a single target region and not replicated. This means if a request takes place between regions, global tokens must be used so that both regions will have the token registered.
+
+# Configuring ACLs
+
+ACLs are not enabled by default, and must be enabled. Clients and Servers need to set `enabled` in the [`acl` stanza](/docs/agent/configuration/acl.html). This enables the [ACL Policy](/api/acl-policies.html) and [ACL Token](/api/acl-tokens.html) APIs, as well as endpoint enforcement.
+
+For multi-region configurations, all servers must be configured to use a single [authoritative region](/docs/agent/configuration/server.html#authoritative_region). The authoritative region is responsible for managing ACL policies and global tokens. Servers in other regions will replicate policies and global tokens to act as a mirror, and must have their [`replication_token`](/docs/agent/configuration/acl.html#replication_token) configured.
+
+# Bootstrapping ACLs
+
+Bootstrapping ACLs on a new cluster requires a few steps, outlined below:
+
+### Enable ACLs on Nomad Servers
+
+The APIs needed to manage policies and tokens are not enabled until ACLs are enabled. To begin, we need to enable the ACLs on the servers. If a multi-region setup is used, the authoritiative region should be enabled first. For each server:
+
+1. Set `enabled = true` in the [`acl` stanza](/docs/agent/configuration/acl.html).
+1. Set `authoritative_region` in the [`server` stanza](/docs/agent/configuration/server.html).
+1. For servers outside the authoritative region, set `replication_token` in the [`acl` stanza](/docs/agent/configuration/acl.html).
```

Comment:
I don't think it is very clear how you get this token. Should it be the bootstrap token? Should it be a new management token specifically dedicated to this. etc.